Extension/src/LanguageServer/configurations.ts

```diff
@@ -846,7 +846,7 @@ export class CppProperties {
             if (isGlobPattern) {
                 // fastGlob silently strips non-found paths. Limit that behavior to dynamic paths only.
                 const matches: string[] = fastGlob.isDynamicPattern(normalized) ?
-                    fastGlob.sync(normalized, { onlyDirectories: true, cwd, suppressErrors: true, deep: 15 }) : [res];
+                    await fastGlob.async(normalized, { onlyDirectories: true, cwd, suppressErrors: true, deep: 15 }) : [res];
                 resolvedGlob.push(...matches.map(s => s + suffix));
```

Comment:
I'm seeing TypeScript fail with 

```
RangeError: Maximum call stack size exceeded
    at CppProperties.resolveAndSplit
```

on line 850 when this is >146k.

```TypeScript
resolvedGlob.push(...matches.map(s => s + suffix));
```

Reply:
I don't have your repro but pushed a change that might handle it. Could you try again?